try
	x=0
catch as error
	if TRUE
		throw error
